kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/common/sqlbuilder/PipelineImportSQLBuilder.java

```diff
@@ -44,12 +46,20 @@ public final class PipelineImportSQLBuilder {
     
     private final PipelineSQLSegmentBuilder sqlSegmentBuilder;
     
-    private final ConcurrentMap<String, String> sqlCacheMap;
+    private final Cache<String, String> sqlCache;
     
-    public PipelineImportSQLBuilder(final DatabaseType databaseType) {
+    public PipelineImportSQLBuilder(final DatabaseType databaseType,final ShardingCacheOptionsConfiguration cacheOptions) {
```

Comment:
Could we define a suitable `cacheOptions` in `PipelineImportSQLBuilder`? This is different with ShardingRouteCache

Reply:
This confused me a lot, but I had two thoughts:
1. Specify parameters directly in the constructor without using cacheOptions
2. Use cacheOptions, but many classes also need to modify methods
So I prefer 1.

Reply:
That's OK. I prefer solution one too.